Source/WebCore/css/parser/CSSTokenizerInputStream.h

```diff
@@ -85,6 +85,7 @@ class CSSTokenizerInputStream {
     }
 
     void advanceUntilNonWhitespace();
+    void advanceUntilNonWhitespaceOrNewline();
```

Comment:
This is the wrong naming. Newlines are whitespace, so the `advanceUntilNonWhitespace` should have the new behavior. The name `advanceUntilNonWhitespaceOrNewline` is ambiguous. Is it "until non-whitespace or newline" or "until non-(whitespace or newline)". You can see that the normal English grammar sense is not what the function does. The new function should be `advanceUntilNewlineOrNonWhitespace`.